server/src/main/java/com/google/coffeehouse/servlets/CreateClubServlet.java

```diff
@@ -76,6 +80,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       Map clubInfo = gson.fromJson(requestBody, Map.class);
       newClub = Club.fromMap(clubInfo, idGen);
     } catch (Exception e) {
+      System.out.println(LOG_BODY_ERROR_MESSAGE + e.getMessage());
```

Comment:
nit: Consider doing e.printStackTrace() instead, which will give you the error message but also the stack trace that led to it.

Reply:
I agree that printing the stack trace would lead to better logs. I avoided doing so in this instance because, upon running our tests, the printed stack trace is easily confused for a real error in the tests (both are printed to stdout during the testing sequence). I will look into finding a way to avoid that issue whilst still logging the full stack trace.